### ISSUE TITLE
Does not support Korean characters

### ISSUE THREAD

**Issue by CoffeePatrol** [Exslims/MercuryTrade#368]
The app doesnt have support for korean characters, it just shows squares instead.

**Comment by Xebtria** [comment]
agree. very annoying considering that perceived half of all trades are happening with korean players since this league.

**Comment by agopwns** [comment]
We used MercuryTrade Korean ver. Do you have any plans to officially support Korean?

**Comment by jcm2606** [comment]
I think I'm getting hit with this bug. Whenever I invite someone with any characters in their name that MercuryTrade doesn't like, they're automatically kicked from the party the moment they accept, without me doing anything. I can invite manually, but the moment I touch MercuryTrade, they're kicked. Very annoying, happened to me a few times the last couple days, and had one guy outright decline my trade because he kept getting kicked due to this bug, and thought it was me doing it.

**Comment by zewt** [comment]
Is this a font problem, or a Windows codepage problem?  I assume Java is Unicode internally so it's probably just the font.  It's a pain buying from Korean users, since you end up with several similar trades all to "☐☐☐☐", "☐☐☐☐☐", "☐☐☐☐☐" and "☐☐☐☐".  Even though MT can't see incoming whispers (so it's limited for trading with them anyway), it would still be useful for remembering which offers are which if this was fixed.
